src/mappings/omim.sssom.tsv

```diff
@@ -48,6 +48,7 @@ MONDO:0000158		skos:exactMatch	OMIMPS:142700	Developmental dysplasia of the hip
 MONDO:0000159		skos:exactMatch	OMIMPS:614675	Bone marrow failure syndrome	semapv:UnspecifiedMatching
 MONDO:0000160		skos:exactMatch	OMIMPS:601068	Epilepsy, familial adult myoclonic	semapv:UnspecifiedMatching
 MONDO:0000166		skos:exactMatch	OMIMPS:610551	Encephalopathy/encephalitis, acute, infection-induced	semapv:UnspecifiedMatching
+MONDO:0000169		skos:exactMatch	OMIMPS:156850		semapv:UnspecifiedMatching
```

Comment:
A page for OMIMPS:156850, which I believe expands out to https://www.omim.org/phenotypicSeries/PS156850, does not exist.

Reply:
The answer is that they are coming straight from Mondo, not from the source.

```make
mondo_exactmatch_omim.sssom.tsv:
	wget "http://purl.obolibrary.org/obo/mondo/mappings/mondo_exactmatch_omim.sssom.tsv" -O $@

omim.owl: omim.ttl mondo_exactmatch_omim.sssom.owl mondo_exactmatch_omimps.sssom.owl
```

So there is no new bug that should cause use to not merge this build.

But it does beg the question, what we should do about this. Probably a GH issue is merited.

Reply:
Agree, not an issue for the build specifically and should not prevent next steps. I can add a ticket.